.github/workflows/maven.yml

```
@@ -423,7 +423,14 @@ jobs:
       - name: Run benchmark
         # Note: user/pass should match those used in sample configs.
         run: |
-          java -jar benchbase.jar -b ${{matrix.benchmark}} -c config/sqlserver/sample_${{matrix.benchmark}}_config.xml --create=true --load=true --execute=true --json-histograms results/histograms.json
+          # For templated benchmarks, we need to preload tpcc data for the test since by design, templated benchmarks do not support the 'load' operation
```

```suggestion
          # For templated benchmarks, we need to preload some data for the test since by design, templated benchmarks do not support the 'load' operation
          # In this case, we load the tpcc data.
```

Is this an accurate adjustment?

---

Yep!